ocs_ci/utility/ibmcloud.py

```diff
@@ -203,3 +205,232 @@ def add_deployment_dependencies():
+
+    def get_data_volumes(self):
+        """
+        Get volumes in IBM Cloud.
+
+        """
+        logger.info("get data volumes")
+
+        cmd = "ibmcloud is vols --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        vol_ids = []
+        for vols in out:
+            vol_ids.append(vols["id"])
+
+        logger.info(f"volume ids are : {vol_ids}")
+        return vol_ids
+
+    def wait_for_volume_attach(self, volume):
+        """
+        Checks volume is attached to node or not
+
+        Args:
+            volume (Volume): The volume to wait for to be attached
+
+        Returns:
+            bool: True if the volume has been attached to the
+                instance, False otherwise
+
+        """
+        cmd = f"ibmcloud is volume {volume} --output json"
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        if not out["volume_attachments"]:
+            logger.info("volume is not attached to node")
+            return False
+        else:
+            logger.info("volume is attached to node")
+            return True
+
+    def getVolumeId(self):
+        """
+        Returns Volumeid with the name taken from constants
+
+        Returns:
+            str: volume id if the volume exists otherwise create
+                new volume
+
+        """
+        zone = config.ENV_DATA["zone"]
+
+        cmd = "ibmcloud is vols --output json "
+        out = run_cmd(cmd)
+        out = json.loads(out)
+
+        vol_id = ""
+        for vols in out:
+            vol_name = str(vols["name"]).strip()
+            if vol_name == constants.IBMCLOUD_VOLUME_NAME:
+                vol_id = vols["id"]
+                return vol_id
+
+        if len(vol_id) == 0:
+            cmd = f"ibmcloud is volume-create {constants.IBMCLOUD_VOLUME_NAME} general-purpose {zone} --output json"
+            out = run_cmd(cmd)
+            out = json.loads(out)
+            vol_id = out["id"]
+            return vol_id
+
+    def restart_nodes_by_stop_and_start(self, nodes, force=True):
+        """
+        Reboot the node on IBM Cloud.
+
+        """
+        logger.info("restart_nodes_by_stop_and_start:")
+        provider_id = nodes[0].get()["spec"]["providerID"]
+        cluster_id = provider_id.split("/")[5]
+
+        cmd = f"ibmcloud ks workers --cluster {cluster_id} --output json"
+        out = run_cmd(cmd)
+        worker_nodes = json.loads(out)
+
+        worker_nodes_not_ready = []
+        for worker_node in worker_nodes:
+            if worker_node["health"]["message"] != "Ready":
+                worker_nodes_not_ready.append(worker_node["id"])
+
+        if len(worker_nodes_not_ready) > 0:
+            for not_ready_node in worker_nodes_not_ready:
+                cmd = f"ibmcloud ks worker reboot --cluster {cluster_id} --worker {not_ready_node} -f"
+                out = run_cmd(cmd)
+                logger.info(f"Node restart command output: {out}")
+
+    def deleteVolumeId(self, volume):
```

Comment:
Please do not use camel case naming style for methods.
```suggestion
    def delete_volume_id(self, volume):
```

Reply:
Done